### PR TITLE
Tag FITSIO.jl v0.8.5

### PR DESCRIPTION
**Pull request by attobot**
Repository: [JuliaAstro/FITSIO.jl](https://github.com/JuliaAstro/FITSIO.jl)
Release: [v0.8.5](https://github.com/JuliaAstro/FITSIO.jl/releases/tag/v0.8.5)
Travis: [Travis Build Status](https://travis-ci.org/JuliaAstro/FITSIO.jl/builds/194697491)
Diff: [vs v0.8.4](https://github.com/JuliaAstro/FITSIO.jl/compare/990aacb09c87c78159aeee12faf47934189a7d4b...5079c542af0a91daa8da314808fd01286e316cdc)
`requires` vs v0.8.4: 
```diff
--- 0.8.4/requires
+++ 0.8.5/requires
@@ -1,3 +1,3 @@
-julia 0.3
+julia 0.4
 BinDeps 0.3.12
 Compat 0.8.6
```
cc: @kbarbary

### CODE REVIEW

**Comment by tkelman** [comment]
Dropping support for a minor Julia version requires a minor package version bump

File: FITSIO/versions/0.8.5/requires [hunk 1]
```diff
@@ -0,0 +1,3 @@
+julia 0.4
+BinDeps 0.3.12
+Compat 0.8.6
```
Comment (tkelman) [inline FITSIO/versions/0.8.5/requires]: and you need 0.9.5 for `take!` on 0.4

**Comment by tkelman** [comment]
closing in favor of #7653

File: FITSIO/versions/0.8.5/requires (hunk 1 above)
Comment (kbarbary) [inline FITSIO/versions/0.8.5/requires]: Oops. I updated the PR too soon.

Comment (tkelman) [inline FITSIO/versions/0.8.5/requires]: attobot lets you re-tag, if you leave the new PR open, delete the existing tag and re-create a new one with the same version number

**Pull request closed without merging**